bin/dynamic_config/update_dcdo.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+SCRIPT=$(readlink -f $0)
+SCRIPT_PATH=`dirname $SCRIPT`
+FULL_CONFIG_PATH=`readlink -f "$1"`
+
+cd $SCRIPT_PATH/../..
+bundle exec ruby $SCRIPT_PATH/lib/update_dcdo.rb $FULL_CONFIG_PATH
```

Comment:
It seems like all this Bash script is doing is executing the ruby script with a single argument. Is there an issue with setting the Ruby script executable and invoking it directly (the pattern used by all the other /bin scripts in this folder)?